float32 velocity
uint16 sub_device_id
---
bool success